shared-libs/transitions/src/lib/schedules.js

```diff
@@ -10,6 +10,16 @@ const messages = require('../lib/messages');
 const messageUtils = require('@medic/message-utils');
 const mutingUtils = require('../lib/muting_utils');
 
+const isMuted = (patient, place) => {
+  if (patient) {
+    return patient.muted || mutingUtils.isMutedInLineage(patient);
+  }
+
+  if (place) {
+    return place.muted || mutingUtils.isMutedInLineage(place);
+  }
+}
```

Comment:
This feels like it's missing a `return false` after the 2 conditional statements. Is that an impossible state though, where we always have either a patient or place?

Reply:
If we don't explicitly return anything, the returned value will be `undefined`, which is a falsey value. 
So we don't need to explicitly return, unless we want to return a truthy value, which we don't in this case.

Reply:
I suppose that's what gets me in the end, a falsey return over `false`. That turns into a coding standard discussion though, so no need to have here.